# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/font-matching/reftest.list
# Commit: cd5ac5f6ff6e
# Full Hash: cd5ac5f6ff6e626bd69e4d42ff9d1d7078abae03
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-03-01 09:40:29
# Regressor Bug: 1756720
# File Overlap Count: 1
# Description:
#   Bug 1756720 - When using legacy MS Symbol fonts, map character codes from the U+00xx range to the PUA range U+F0xx found in the cmap subtables. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139724
# ==============================================================================

diff -r b547c1a2ef61 -r cd5ac5f6ff6e layout/reftests/font-matching/reftest.list
--- a/layout/reftests/font-matching/reftest.list	Mon Feb 28 20:34:33 2022 +0000
+++ b/layout/reftests/font-matching/reftest.list	Mon Feb 28 20:38:38 2022 +0000
@@ -172,5 +172,9 @@
 # Disabled because Osaka-Mono isn't available on automation.
 # skip-if(!cocoaWidget) == osaka-mono.html osaka-mono-ref.html
 
+# Try to use Wingdings font (assumed present on Windows and macOS)
+random-if(!winWidget&&!cocoaWidget) == wingdings-1.html wingdings-1-ref.html
+random-if(!winWidget&&!cocoaWidget) != wingdings-1.html wingdings-1-notref.html
+
 # Reset default prefs.
 defaults